### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by CPinelli**
Description:

Linting:

- [x] No linting errors

Tests:

- [ ] Manual tests

Documentation:

- [x] Requires documentation updates

### CODE REVIEW

File: chatbot/README.md [hunk 1]
```diff
@@ -146,6 +157,22 @@ You can then click on the order to have a new component popping up:
 At this point the chatbot is up and running!
 
 
+### Docker
```
Comment (shaunmaharaj) [inline chatbot/README.md]: I'd prefer grouping the `Docker` section with the `Deploy the application` section to keep things together if they're related (for production deployments). 
If the `Docker` section is meant for production, lets rename it to `Deploy for production`, similar to the Reference Storefront's layout.

File: login/README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
       * [Configuration Parameter Descriptions](#configuration-parameter-descriptions)
       * [Setting up a Development Environment](#setting-up-a-development-environment)
       * [Setting up a Production Environment](#setting-up-a-production-environment)
+      * [Docker](#docker)
```
Comment (shaunmaharaj) [inline login/README.md]: Same comment regarding the `Docker` step. Can this be merged with `Setting up a Production Environment`?

File: login/README.md [hunk 3]
```diff
@@ -45,7 +48,13 @@ This Builds the app for production to the `build` folder.<br>
 It correctly bundles React in production mode and optimizes the build for the best performance.<br>
 The build is minified and the filenames include the hashes.<br>
 6. The built files may be hosted on any webserver of your choosing.
-6. To see the running Reference Facebook Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
+7. To see the running Reference Facebook Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
+
+### Docker
+The application contains a `Docker` folder that contains itself a Dockerfile, a Shell script and an Nginx configuration file.
+The Dockerfile is using a regular NodeJS image and a regular Nginx image, it uses the Production environment (see above). It contains 2 arguments (`SCOPE` and `CORTEX_URL`) that you can populate using the `--build-arg` argument when creating the Docker image
```
Comment (shaunmaharaj) [inline login/README.md]:
```suggestion
The Dockerfile is using a NodeJS image and an Nginx image. It uses the Production environment (see above). It contains 2 arguments (`SCOPE` and `CORTEX_URL`) that you can populate using the `--build-arg` argument when creating the Docker image
```

File: login/README.md [hunk 4]
```diff
@@ -45,7 +48,13 @@ This Builds the app for production to the `build` folder.<br>
...
+The Shell script is populating the configuration file according to the arguments that are passed to the Docker image. 
+The Nginx configuration just exposes the built files that are created by the Dockerfile when running `npm run build`.
```
Comment (shaunmaharaj) [inline login/README.md:54]:
```suggestion
The Nginx configuration exposes the built files that are created by the Dockerfile when running `npm run build`.
```

File: login/README.md [hunk 5]
```diff
@@ -45,7 +48,13 @@ This Builds the app for production to the `build` folder.<br>
 It correctly bundles React in production mode and optimizes the build for the best performance.<br>
 The build is minified and the filenames include the hashes.<br>
 6. The built files may be hosted on any webserver of your choosing.
-6. To see the running Reference Facebook Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
+7. To see the running Reference Facebook Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
```
Comment (shaunmaharaj) [inline login/README.md]:
```suggestion
7. To see the running Reference Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
```

Comment (shaunmaharaj) [reply login/README.md]: Please rename `Reference Facebook Chatbot` to `Reference Chatbot` in the documentation.

File: login/README.md [hunk 6]
```diff
@@ -26,6 +27,8 @@ You must configure the following parameters in the `./src/ep.config.json` file:
 |`cortexApi.scope`| Required| String| Name of the store from which Cortex retrieves data.|
 |`cortexApi.pathForProxy`|Required|String| The path to which the [Webpack](https://webpack.js.org/) proxy routes the Cortex calls from the storefront. This value is a URL that consists of hostname and port of a running instance of Cortex. Leave this field blank to disable proxy.|
 
+NB: See below for automation
```
Comment (shaunmaharaj) [inline login/README.md]:
```suggestion
```

File: chatbot/README.md [hunk 7]
```diff
@@ -146,6 +157,22 @@ You can then click on the order to have a new component popping up:
...
+
+The application contains a `Docker` folder that contains a Dockerfile and a Shell script. The Dockerfile uses a regular NodeJS image and contains multiple arguments that you can populate using the `--build-args` argument when building the image.
+These Docker arguments are used in the Shell script to replace the constants in the code, this way, the chatbot is easily deployable and doesn't require to modify directly the code.
+
+|  Parameter| Importance|Type|Description|
+|--|--|--|--|
+|`PAGE_ACCESS_TOKEN`| Required| String| The token that is provided by Facebook on (developers.facebook.com) for each application.|
+|`VERIFY_TOKEN`| Required| String| The token that you have to generate and pass to Facebook when configuring the webhooks.|
+|`SCOPE`|Required|String| The Cortex scope (store) that the chatbot is going to use.|
```
Comment (shaunmaharaj) [inline chatbot/README.md]:
```suggestion
|`SCOPE`|Required|String| Name of the store from which Cortex retrieves data.|
```

File: chatbot/README.md [hunk 8]
```diff
@@ -146,6 +157,22 @@ You can then click on the order to have a new component popping up:
...
+|`S3_URL`|Required|String| The AWS S3 URL that the application is using to fetch images.|
```
Comment (shaunmaharaj) [inline chatbot/README.md]:
```suggestion
|`S3_URL`|Required|String| The URL that consists of the path to images hosted on an external CMS.|
```

If possible, let's also try to describe how the url should be structured. Similar to how we describe it for the Reference Storefront.

**Review by shaunmaharaj — CHANGES_REQUESTED** [review]

File: chatbot/README.md (hunk 1 above)
Comment (CPinelli) [reply chatbot/README.md]: Fixed

File: login/README.md (hunk 2 above)
Comment (CPinelli) [reply login/README.md]: Fixed

**Review by shaunmaharaj — APPROVED** [review]

**Pull request merged**